machine/v1beta1/types_gcpprovider.go

```diff
@@ -54,9 +54,20 @@ type GCPMachineProviderSpec struct {
 	// ProjectID is the project in which the GCP machine provider will create the VM.
 	// +optional
 	ProjectID string `json:"projectID,omitempty"`
+	// GuestAccelerators is a list of GPUs to be attached to the VM.
```

Comment:
Are these necessarily GPUs?  "Accelerators" is a broad enough term that it could encompass FPGAs, for example.

Reply:
"Google Cloud Platform provides graphics processing units (accelerators) that you can add to VM instances to improve or accelerate performance when working with intensive workloads. " - from GCP docs

Reply:
> "Google Cloud Platform provides graphics processing units (accelerators) that you can add to VM instances to improve or accelerate performance when working with intensive workloads. " - from GCP docs

If these are explicitly GPUs, it seems worth calling them such in our API.

Reply:
As far as I can tell, the only thing we can add here currently are GPUs. All documentation about accelerator types links back to GPUs. @SamuelStuchly let's rename this field to `GPUs`